pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_space_foi.py

```diff
@@ -12,60 +13,85 @@ class TestSpatialInfection(TestPyEpiabm):
 
     @patch('pyEpiabm.core.Parameters.instance')
     def test_space_susc_no_age(self, mock_params):
+        self.placeclosure(self.time)
         mock_params.return_value.use_ages = False
-        result = SpatialInfection.space_susc(self.cell, self.infectee,
-                                             self.time)
+        result = SpatialInfection.space_susc(
+            self._population.cells[0], self._population.cells[0].
+            microcells[0].persons[0], self._population.cells[0].
+            microcells[0].persons[1], self.time)
         self.assertIsInstance(result, float)
-        self.assertEqual(result, 1)
+        self.assertEqual(result, 0.5)
 
     def test_space_inf(self):
-        result = SpatialInfection.space_inf(self.cell, self.infector,
-                                            self.time)
+        self.placeclosure(self.time)
+        result = SpatialInfection.space_inf(
+            self._population.cells[0], self._population.cells[0].
+            microcells[0].persons[0], self.time)
         self.assertTrue(result > 0)
         self.assertIsInstance(result, float)
 
     @patch('pyEpiabm.core.Parameters.instance')
     def test_space_inf_no_age(self, mock_params):
+        self.placeclosure(self.time)
         mock_params.return_value.use_ages = False
-        result = SpatialInfection.space_inf(self.cell, self.infector,
-                                            self.time)
+        result = SpatialInfection.space_inf(
+            self._population.cells[0], self._population.cells[0].
+            microcells[0].persons[0], self.time)
         self.assertIsInstance(result, float)
-        self.assertEqual(result, self.infector.infectiousness)
+        self.assertEqual(result, 0.5 * self._population.cells[0].
+                         microcells[0].persons[0].infectiousness)
 
     def test_space_foi(self):
-        result = SpatialInfection.space_foi(self.cell, self.cell,
-                                            self.infector, self.infectee,
-                                            self.time)
+        self.placeclosure(self.time)
+        result = SpatialInfection.space_foi(
+            self._population.cells[0], self._population.cells[0],
+            self._population.cells[0].microcells[0].persons[0],
+            self._population.cells[0].microcells[0].persons[1],
+            self.time)
         self.assertTrue(result > 0)
         self.assertIsInstance(result, float)
 
     def test_cell_inf(self):
-        self.infector.update_status(InfectionStatus.InfectMild)
-        result = SpatialInfection.cell_inf(self.cell, self.time)
+        infector = self._population.cells[0].microcells[0].persons[0]
+        infector.update_status(InfectionStatus.InfectMild)
+        result = SpatialInfection.cell_inf(self._population.cells[0],
+                                           self.time)
         self.assertIsInstance(result, float)
         self.assertTrue(result >= 0)
 
```

Comment:
Make a function that tests if the spatial infectiousness changes after including case isolation. See test written for case isolation